http/vibe/http/fileserver.d

```diff
@@ -311,7 +312,41 @@ private void sendFileImpl(scope HTTPServerRequest req, scope HTTPServerResponse
 	if ("Content-Encoding" in res.headers && isCompressedFormat(mimetype))
 		res.headers.remove("Content-Encoding");
 	res.headers["Content-Type"] = mimetype;
-	res.headers["Content-Length"] = to!string(dirent.size);
+	res.headers.addField("Accept-Ranges", "bytes");
+	ulong rangeStart = 0;
+	ulong rangeEnd = 0;
+	auto prange = "Range" in req.headers;
+
+	if (prange) {
+		auto range = (*prange).chompPrefix("bytes=");
+		auto s = range.split("-");
+		// https://tools.ietf.org/html/rfc7233
+		// Range can be in form "-\d", "\d-" or "\d-\d"
```

Comment:
"\" - not sure if that's just my browser, but that's supposed to be a backslash.